addons/test_mail_full/tests/test_rating.py

```diff
@@ -159,20 +175,38 @@ def test_rating_last_value_perfs(self):
                 record.rating_apply(5, token=access_token)
             self.flush_tracking()
 
-        with self.assertQueryCount(employee=1):
-            record_ratings._compute_rating_last_value()
-            vals = [val == 5 for val in record_ratings.mapped('rating_last_value')]
-            self.assertTrue(all(vals), "The last rating is kept.")
+        if is_rating_mixin_test:
+            with self.assertQueryCount(employee=1):
+                record_ratings._compute_rating_last_value()
+                vals = [val == 5 for val in record_ratings.mapped('rating_last_value')]
+                self.assertTrue(all(vals), "The last rating is kept.")
+
+    @users('employee')
+    @warmup
+    def test_rating_last_value_perfs(self):
+        self._test_rating_last_value_perfs('mail.test.rating.without.mixin', 'partner_id', False)
+
+    @users('employee')
+    @warmup
+    def test_rating_last_value_perfs_with_rating_mixin(self):
+        self._test_rating_last_value_perfs('mail.test.rating', 'customer_id', True)
 
 
 @tagged('rating')
 class TestRatingRoutes(HttpCase, TestRatingCommon):
 
-    def test_open_rating_route(self):
-        access_token = self.record_rating._rating_get_access_token()
+    def _test_open_rating_route(self, record_rating, is_rating_mixin_test):
+        access_token = record_rating._rating_get_access_token()
         self.url_open(f"/rate/{access_token}/5")
 
-        rating = self.record_rating.rating_ids
+        rating = record_rating.rating_ids
         self.assertTrue(rating.consumed)
         self.assertEqual(rating.rating, 5)
-        self.assertEqual(self.record_rating.rating_last_value, 5)
+        if is_rating_mixin_test:
+            self.assertEqual(record_rating.rating_last_value, 5)
+
+    def test_open_rating_route(self):
```

Comment:
Same here, we could start using subTests to make tests clearer.

Reply:
Done too, thanks.